data/ut/legislature/Daniel-W-Thatcher-dd3219ba-dd82-44a1-b3e5-27c17fd55a2d.yml

```diff
@@ -0,0 +1,36 @@
+id: ocd-person/dd3219ba-dd82-44a1-b3e5-27c17fd55a2d
+name: Daniel W. Thatcher
+email: dthatcher@le.utah.gov
+image: https://le.utah.gov/images/legislator/THATCDW.jpg
+party:
+- name: Republican
+roles:
+- start_date: '2019-01-01'
+  type: upper
+  jurisdiction: ocd-jurisdiction/country:us/state:ut/government
+  district: '11'
+offices:
```

Comment:
@sroomf this person should already have a file and we should add their new position:
- start_date: '2023-01-01'
  type: upper
  jurisdiction: ocd-jurisdiction/country:us/state:ut/government
  district: '11'
  
 They've served in the senate district 12 from Jan 2011-Jan 2023